### PR TITLE
Add "Value" and "Datatype" to the Glossary

### PR DESCRIPTION
**Pull request by Silvan-WMDE**

### CODE REVIEW

**Review by addshore — APPROVED** [review]

File: systems/overview/12-Glossary.md [hunk 1]
```diff
@@ -107,6 +111,10 @@ Tools which help when working with Wikidata. The full list of Wikidata tools can
 
 A textual description of the functionality provided by the system that captures actor-system interaction. It specifies how a user interacts with a system and how the system responds to the user actions.
 
+## Value
```
Comment (addshore) [inline systems/overview/12-Glossary.md:114]: Per a discussion in mattermost we should not refer to this as a "value" only a "datavalue".
per https://www.mediawiki.org/wiki/Wikibase/DataModel#Values

OR we should remove this other definition of "value" from docs.

**Review by addshore — CHANGES_REQUESTED** [review]

File: systems/overview/12-Glossary.md (hunk 1 above)
Comment (jakobw) [reply systems/overview/12-Glossary.md:114]: > Per a discussion in mattermost we should not refer to this as a "value" only a "datavalue".
per https://www.mediawiki.org/wiki/Wikibase/DataModel#Values
> 
> OR we should remove this other definition of "value" from docs.

As far as I know "value" / "statement value" is what it's more commonly referred to. I haven't heard anyone use "datavalue" as a business term.

Comment (Silvan-WMDE) [reply systems/overview/12-Glossary.md]: Fixed by moving Value to Datavalue and changing all links to it. I still believe, we should fix the docs at https://www.mediawiki.org/wiki/Wikibase/DataModel#Values. I now refer the Glossary entry to https://www.wikidata.org/wiki/Help:Statements#Values, which says very different things about what _Values_ are.

**Review by jakobw — APPROVED** [review]
I'm still not entirely convinced "datavalues" is more commonly used than "(statement) values" in the language of product people or users. The contents make sense to me though :+1:

**Pull request merged**